samples/.eslintrc.yml

```diff
@@ -1,3 +1,5 @@
 ---
+env:
```

Comment:
We could move this down into `samples/system-test`

Reply:
Sure , done